build.sh

```diff
@@ -1,3 +1,20 @@
 #!/bin/sh
 
-docker build -t outrigger/node:4 .
+set -e
+
+# Get the versions to build
+
+# If they are passed as args
+if [ $# -gt 0 ]; then
+  buildDirs="$@"
+else
+  # else, get all the relevant dirs
+  buildDirs=$(ls -1 | grep -E '^[0-9]')
```

Comment:
This will build any directory that starts with a number. The idea was to support having an 8-build, but I'm not sure that needs an explicit image now that I've added git to the main image.